### DISCUSSION TITLE
Работа с инвентарем.

### DISCUSSION THREAD

**Discussion by stalkersof** [OGSR/OGSR-Engine#499] [Q&A] [answered]
Можно как то получить текущий вес в инвентаре? Вес всех предметов у актора в рюкзаке слотах и на поясе. Есть какой нибудь метод?

**Comment by PRXPHET** [comment, marked as answer]
https://github.com/OGSR/OGSR-Engine/blob/9abb2f107fa09fb358832d4c273149449575053a/ogsr_engine/xrGame/script_game_object_script3.cpp#L273
